Safety/Enter-Observatory-Spaces/Enter-AuxTel-Telescope.rst

```diff
@@ -0,0 +1,190 @@
+
+
+   .. figure:: ./_static/keys.png
+       :name: keys
+       :scale: 25 %
+
+       Key hook
+
+
+
+   .. note:: 
+       Exceptionally, the entrance door will currently be left unlocked.
+
+
+
+3. **Communication Protocols**
+   
+   - Announce your visit to the telescope indicating the reasons briefly via the Slack channels `#summit-auxtel <https://lsstc.slack.com/archives/C01K4M6R4AH>`__ and `#summit-announce <https://app.slack.com/client/T06D204F2/C01P41NUR1R>`__.
+
+   - Communicate by radio on channel 3 when entering the telescope during the day and night. This action is suggested.
+  
+
+.. _Safe-entry-to-AuxTel-access-during-ventilation:
+
+Access During Ventilation
+=========================
+
+
+.. figure:: ./_static/vent.png
+    :name: Venting
+
+    Air extraction fan.
+
+
+
+
+1. **Ventilation Notification**
+
+   - Ventilation starts with a notification via Slack channels and/or radio channel 3.
+
+
+
+2. **Safety Precautions**
+
+   - Ensure the windows are fully open and the fan operates at 20%.
+
+   - Take special care not to lean on the windows or manipulate the fan while it is running.
+
+   - On the telescope's second floor, verify that the telescope is in a horizontal position and the dome is facing opposite the Sun to maximize the ventilation system.
+
+
+
+.. _Safe-entry-to-AuxTel-access-during-calibrations:
+
+Access During Calibrations
+==========================
+
+.. warning::
+
+    Access is **NOT** permitted during calibrations unless there is an issue with the system that requires attention, and images can be interrupted or are not currently being taken. 
+    Ensure there are no announcements in Slack, and confirm with summit staff (daytime shift workers or an Observing Specialist) that no images are in progress.
+
+
+
+.. _Safe-entry-to-AuxTel-access-during-night-time-operations:
+
+Access During Night time Operations
+===================================
+
+1. **Entry Permissions**
+
+   - Only authorized personnel and the Observing Specialist team can enter without an external permit during operations.
+
+
+2. **Lighting Restrictions**
+   
+   .. warning:: 
+
+    During operations, it is prohibited to turn on building lights when entering the telescope.
+
+
+   .. caution:: 
+        
+    The use of flashlights or headlamp is permitted but they should point towards the ground and should be minimized to avoid interfering with observations.
+
+
+   .. tip:: 
+
+    Red light flashlights can be a good choice and they interfere much less than regular white light.
+
+
+.. _Safe-entry-to-AuxTel-behaviour-standards:
+
+Behavior Standards
+==================
+
+Maintain proper behavior at all times, including:
+
+- Being close to a radio (or always carrying one) and communicating your intentions to summit staff and on Slack.
+  
+- Following the `safety gate procedure <https://obs-ops.lsst.io/v/SITCOM-1360/Safety/Safety-Systems/AuxTel-Safety-Gate-Introduction.html>`__.
```

Comment:
Replace this external reference with an internal tag to the Safety Gate document. It will start working once both PR get merged.

Reply:
This link won't work once SITCOM-1360 is merged nor do we want to link to an https, that might in the future give make linkcheck problems ;)